### DISCUSSION TITLE
Change yellow color

### DISCUSSION THREAD

**Discussion by stardepp** [Open-Shell/Open-Shell-Menu#1066] [Q&A] [answered]
How can I change the highlight color?

![image](https://user-images.githubusercontent.com/40627059/175797573-99446536-06e5-44e1-88ef-c6d71971ecfb.png)

**Comment by blackcrack** [comment]
in dem Du im Windows unter erweiterte Darstellung
die Farbe änderst ?! Oder Du benutzt einen anderen Skin.. im OpenShell
Ich benutz "Smoked Glass" da brauch ich keine Farbe.. unter "Skin" und unter "Skin f. alle Programme"
weitere Skins kannst Du noch unter classicshell.net runter laden.. 
auch die language-bezogene dll's, die du einfach in das programmverzeichniss schmeisst,
oder schausst unter den Einstellungen unter "Der Erweiterten Ansicht" nach ob das überhaupt geht..
Deiner Frage nach, hast Du nich mal richtig geschaut..
Mit welcher Version arbeitest Du überhaupt ?
Wir haben hier einen Hauptprogrammierer, einen der gern mal was dazu baut und ein bisschen überwacht
und einen, Ibuprophen , der hier Mithilft im englishen..
keiner wird bezahlt, die Programmierer haben kaum Zeit und Ibuprophen als Teammember 
kann nich Programmieren iss aber echt voll korrekt..

Wenn Du aber mithelfen willst, schau nach jemandem, der Programmieren kann
und mithilft kann das Proggy auf Win11 sauber zum laufen zu bekommen,
das iss jetzt erstmal wichtig..

danke und viele Grüße
Blacky

**Comment by stardepp** [comment]
Sorry. Thanks for your answer.

I use Windows 10 and in the color settings in Windows I have set gray. I use the open shell skin Metro. But no matter which skin I choose, it stays with the yellow color.

**Comment by blackcrack** [comment]
i guess it's the configuration in Windows.. your color settings.. und wenn de schon ein deutsches Menu hast und auf Deutsch beantwortet wird, dann kannste auch auf Deutsch antworten ;)
Iss ja nich so, daß wir Chinesen währen *kicher*

**Comment by stardepp** [comment]
Ok, aber wo in den Windows Einstellungen soll das denn sein?

**Comment by CTVCAM8** [comment, marked as answer]
Each skin has a different way of showing the new program's highlighted color.

The "Windows Aero" skins use a combination of orange text and orange selector bitmap images.

The "Metro" skins use a yellow Metro text color, as well as text and bitmap images that mix 50% of the yellow color with the menu color. 


At this time, the new program's color can only be changed by editing the skin file.
So, in your case the Metro.skin7 file would need to be altered using a program called "Resource Hacker".

It's best to make of copy of the Metro.skin7 file located in C:\Program Files\Open-Shell\Skins folder
and put on the desktop, as well as rename it to something like "Metro custom new programs color.skin7"

Open that file with Resource Hacker, and look in SKIN folder 1 to see text and numbers. 
These are the lines of text that need to be changed for a different color. 

Main_new_selection_tint1=$StartInlineErrorText
Main2_new_selection_tint1=$StartInlineErrorText
Programs_new_selection_tint1=$StartInlineErrorText
Submenu_new_selection_tint1=$StartInlineErrorText


Option 1. 
To get a blue color, change text to:

Main_new_selection_tint1=#0096FF
Main2_new_selection_tint1=#0096FF
Programs_new_selection_tint1=#0096FF
Submenu_new_selection_tint1=#0096FF

Hit the "Compile Script" button.
Save the modified skin and put it in the skins folder. Pick the modified skin in the skin dropdown list in the Open-Shell settings.




Or Option 2. Color picker added. 
Leave the original text the way it is, and add special below text at the very bottom of the text file so it overrides the yellow color:

OPTION_COLOR CUSTOM_NEW_PROGRAMS_COLOR=Custom new programs color,0,TRUE,FF9600

[CUSTOM_NEW_PROGRAMS_COLOR]
Main_new_selection_tint1=@CUSTOM_NEW_PROGRAMS_COLOR@
Main2_new_selection_tint1=@CUSTOM_NEW_PROGRAMS_COLOR@
Programs_new_selection_tint1=@CUSTOM_NEW_PROGRAMS_COLOR@
Submenu_new_selection_tint1=@CUSTOM_NEW_PROGRAMS_COLOR@

Hit the "Compile Script" button.
Save the modified skin and put it in the skins folder. Pick the modified skin in the skin dropdown list in the Open-Shell settings.

Just remember the chosen color is mixed with the start menu color at 50% mix, so it will look duller. 
To get 100% true color would mean extra text and bitmap images would need to be modified. 
Best to look at the skinning tutorial for more information:

http://classicshell.net/tutorials/skintutorial.html

![Metro custom  new programs color](https://user-images.githubusercontent.com/44127216/179912175-3128d92a-3adb-444e-946a-f1454bf5c488.png)


Extra note:

Just in case someone wants the color picker on the Metro.skin (classic 1 and 2 column styles),
additional text to unhide the color picker option needs to be added.
Look for lines 164 and 165 in Resource Hacker. Add ", CUSTOM_NEW_PROGRAMS_COLOR" at the end of each line so it looks like this:

Classic1_options=CAPTION, USER_IMAGE, USER_NAME, CENTER_NAME, SMALL_ICONS, LARGE_FONT, ICON_FRAMES, OPAQUE, CUSTOM_NEW_PROGRAMS_COLOR
Classic2_options=NO_ICONS, USER_IMAGE, USER_NAME, CENTER_NAME, SMALL_ICONS, LARGE_FONT, ICON_FRAMES, OPAQUE, CUSTOM_NEW_PROGRAMS_COLOR

**Reply by stardepp** [reply]
Thank you so much for your professional help, it worked so well.

**Reply by blackcrack** [reply]
der macht das nich beruflich :) das iss ne persönliche hilfe ;)
sach mal, das müsste doch gehen über die Farbeinstellung in Rechte maustaste, anpassen -> Fensterfarbe -> erweitert -> Farbenfenster und Darstellung und da das ausgewählte Element.. benutz selber noch nur Win7 und Fedora, deshalb bin ich grad nich so aktuell mit win10/11..
Aber dass man das direkt im Theme einstellen kann iss auch ned schlecht.. sachmal ne kleine dezente frage (meins nich bös), hättest Du dir nich nen besseren Alias nehmen können \*dichangrinz* Aaaalllttter \*smile*
